identity-service/test/emailNotifications/renderedEmails/additionalChallengeEmail.html

```diff
@@ -592,4 +592,4 @@
><td valign="center" style="margin:0px"><img src="https://download.audius.co/static-resources/email/iconTwitterWhite.png" alt="twitter" style="height:18px;width:18px;padding:0px;margin-right:8px;vertical-align:text-bottom"/></td><td valign="center" style="margin:0px;font-size:14px;font-weight:500;color:#ffffff;text-decoration:none">Share this with your fans</td></tr></table></td></tr></table></a></td></tr><tr><td valign="center" colspan="12" style="padding:0px 16px 14px"><a class="avenir" style="width:337px;color:#C2C0CC;font-size:12px;font-weight:600;line-height:10px">Open Audius<img src="https://download.audius.co/static-resources/email/iconArrow.png" style="height:8px;width:8px;margin-left:4px;display:inline-block" alt="Arrow Right" titile="Arrow Right" class="arrowIcon"/></a></td></tr></table></a></td></tr></table></td></tr><tr><td align="center" valign="top" id="bodyCell" style="padding:24px 0px 32px;width:100%"><table cellspacing="0" cellpadding="0" style="margin:0px auto"><tr><td style="border-radius:17px;margin:0px auto" bgcolor="#7E1BCC"><a href="https://audius.co/feed?openNotifications=true" target="_blank" style="padding:8px 24px;font-size:14px;color:#ffffff;text-decoration:none;font-weight:bold;display:inline-block">See more on Audius</a></td></tr></table></td></tr><tr><td style="padding-bottom:25px"><table border="0" cellpadding="0" cellspacing="0" style="margin:0px auto;height:auto;padding-botton:25px"><tr><td valign="center" id="socialBar" style="text-align:center;padding:25px 0px 20px"><a href="https://www.instagram.com/audiusmusic/"><img src="https://download.audius.co/static-resources/email/iconInsta.png" alt="instagram" style="height:24px;width:24px;padding:0px 24px"/></a><a href="https://twitter.com/AudiusProject"><img src="https://download.audius.co/static-resources/email/iconTwitter.png" alt="twitter" style="height:24px;width:24px;padding:0px 24px"/></a><a href="https://discordapp.com/invite/yNUg2e2"><img src="https://download.audius.co/static-resources/email/iconDiscord.png" alt="discord" style="height:24px;width:24px;padding:0px 24px"/></a></td></tr><tr><td valign="center" style="text-align:center;padding:0px 0px 8px;margin:0px"><div class="gilroy" style="text-align:center;color:#858199;font-size:14px">Made with <span style="color:#7E1BCC">♥︎</span> in SF &amp; LA</div></td></tr><tr><td style="text-align:center;vertical-align:center;height:auto;padding:0px 0px 12px;margin:0px"><div class="gilroy" style="text-align:center;color:#858199;font-size:14px">© <!-- -->2022<!-- --> Audius, Inc. All Rights Reserved.</div></td></tr><tr><td valign="top" id="utilityBar"><div class="gilroy" style="text-align:center;color:#858199;font-size:14px">Tired of seeing these emails? <a href="https://audius.co/settings" class="utilityLink" style="text-decoration-color:#858199"><span style="color:#858199">Update your notification preferences</span></a><span class="mobileHide"></span></div></td></tr></table></td></tr></table></center></body></html>
```

Comment:
I'm assuming this is just updated to have the two new email bodies for the new challenges? My eyes are bleeding trying to parse it so I'm just going to assume it's good :eyes: :joy:

Reply:
Yes I also just copy-pasted the `actual` output from the failed test tbh, but I did confirm it had the two new challenges in there as well as the old ones. @sddioulde not sure if you have a better way of doing this?